client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-overview/style.scss

```diff
@@ -0,0 +1,110 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.payment-method-overview {
+	@include breakpoint-deprecated( "<660px" ) {
+		.jetpack-cloud-layout__header-main {
+			margin-block-end: 16px;
+		}
+
+		.jetpack-cloud-layout__header-actions .button {
+			width: 100%;
+		}
+	}
+}
+
+.payment-method-overview-empty-state {
+	margin-block-start: 60px;
+
+	img {
+		margin: auto;
+		display: block;
+		width: 270px;
+	}
+}
+
+.payment-method-overview-empty-state__top-content {
+	width: 315px;
+	margin: auto;
+
+	.button {
+		margin-block-start: 40px;
+		width: 100%;
+	}
+}
+
+.payment-method-overview-empty-state__bottom-content {
+	margin-block-start: 50px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+
+	.payment-method-overview-empty-state__card {
+		flex: 1;
+		width: calc(33.33% - 64px); /* 64px is the total margin width */
+		margin: 16px;
+		padding: 24px;
+		box-sizing: border-box;
+		flex-basis: 100%;
+		text-align: center;
+
+		svg {
+			fill: var(--color-neutral-40);
+			margin-block-end: 24px;
+		}
+
+		.payment-method-overview-empty-state__card-title {
+			font-size: rem(18px);
+			font-weight: 600;
+			margin-block-end: 8px;
+		}
+
+		.payment-method-overview-empty-state__card-description {
+			color: var(--color-neutral-60);
+			font-size: rem(14px);
+		}
+
+		@include break-medium {
+			flex-basis: calc(50% - 10px);
+		}
+
+		@include break-large {
+			flex-basis: unset;
+		}
+	}
+}
+
+.payment-method-overview__stored-cards {
+	margin-block-start: 50px;
+	display: flex;
+	flex-wrap: wrap;
+	gap: 24px;
+
+	@include breakpoint-deprecated( "<660px" ) {
+		margin-block-start: 32px;
+	}
```

Comment:
Should we continue using this deprecated breakpoint?

I think we should take advantage of this porting from Manage to A4A to fix it wherever we can. We can use `break-small` (from `@wordpress/base-styles/breakpoints`) or `$break-small` (from `@wordpress/base-styles/mixins`) instead of it. It has a different breaking point, `600px` instead of `660px`. I don't know how it'd look.

Reply:
That is a good point. This has been addressed. Thanks for the feedback. 👍